bot/admin/web/src/app/applications/applications.module.ts

```diff
@@ -84,7 +82,6 @@ export class NlpApplicationConfig implements ApplicationConfig {
     NbInputModule,
     NgJsonEditorModule,
     NbDialogModule.forRoot(),
-    NgbPaginationModule,
```

Comment:
Is this module replaced by an other pagination module ?

Reply:
Yes, on the bot module we have been using a homemade one for several months. This was the last component that forbade us to drop the ng-bootstrap dependency (we use bootstrap but only the css part of it)